x-pack/test/ingest_manager_api_integration/apis/epm/file.ts

```diff
@@ -47,7 +47,6 @@ export default function ({ getService }: FtrProviderContext) {
             '/api/ingest_manager/epm/packages/filetest/0.1.0/kibana/visualization/sample_visualization.json'
           )
           .set('kbn-xsrf', 'xxx')
-          .expect('Content-Type', 'text/plain; charset=utf-8')
```

Comment:
@jfsiii @skh @ruflin I'm removing these checks because it seems that they change based on the base docker container we use or between releases of the package-registry.

If I hit these apis chrome returns `application/json`:
https://epr.elastic.co/package/aws/0.1.1/kibana/dashboard/0eb5a6a0-694f-11ea-b0ac-95d4ecb1fecd.json

Same for:
https://epr-staging.ea-web.elastic.dev/package/aws/0.1.1/kibana/dashboard/0eb5a6a0-694f-11ea-b0ac-95d4ecb1fecd.json

And via the ingest manager api it's `application/json` too:

![image](https://user-images.githubusercontent.com/56361221/88830439-f512af80-d19b-11ea-84ce-08412624c17e.png)

Reply:
Skipping the tests based on Sonja's guidance.